.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+      - master  # Trigger only on push to master branch (e.g., after merging a PR)
+  pull_request:
+    branches: 
+      - master
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)